main.py

```diff
@@ -109,7 +109,25 @@ def in_danger(self, w, h):
         else:
             return False 
 
-    
+    def get_dust_amount(self, i, j):
```

Comment:
Can you get rid of get_dust_amount and is_tile_cleaned because I no longer have those in my code. If you get rid of those I can just merge the changes and won't have to update anything besides seeing how we can check that stats stay under 10. Aside from that this looks super cool and I'm excited to run it.

Reply:
I had to add these in order for main.py to work. I cloned you repo to add my changes and when I tried to run main.py it said those functions weren't added so it didn't work

Reply:
Oh okay, I'll merge this then